docs/api/IoT.js-API-ADC.md

```diff
@@ -10,47 +10,28 @@ The following shows ADC module APIs available for each platform.
 | adcpin.close | O | O | O |
 | adcpin.closeSync | O | O | O |
 
-
-## Contents
-* [Pin](#pin)
-* [ADC](#adc)
-  * [Constructor](#adc-constructor)
-    * [`new ADC()`](#adc-new)
-  * [Prototype methods](#adc-prototype-methods)
-    * [`adc.open(configuration[, callback])`](#adc-open)
-* [ADCPin](#adcpin)
-  * [Prototype methods](#adcpin-prototype-methods)
-    * [`adcpin.read([callback])`](#adcpin-read)
-    * [`adcpin.readSync()`](#adcpin-read-sync)
-    * [`adcpin.close([callback])`](#adcpin-close)
-    * [`adcpin.closeSync()`](#adcpin-close-sync)
-
-## `pin` <a name="pin"></a>
-On Nuttx, you have to know pin number that is defined in target board module. For more module information, please see below list.
+### `pin`
+On Nuttx, you have to know the number of pins that is defined on the target board module. For more information, please see the list below.
```

Comment:
I would put this text under the `## Class: ADC`, remove the `### pin` header, and add a bit more text about what this component does.